controller/controller.go

```diff
@@ -1235,8 +1235,10 @@ func (ctrl *ProvisionController) shouldProvision(ctx context.Context, claim *v1.
 // shouldDelete returns whether a volume should have its backing volume
 // deleted, i.e. whether a Delete is "desired"
 func (ctrl *ProvisionController) shouldDelete(ctx context.Context, volume *v1.PersistentVolume) bool {
+	klog.V(4).Infof("shouldDelete Volume : %+v", volume)
```

Comment:
Probably use level 5 for debugging logs.

Reply:
I thought we were supposed to use level 4 for debugging logs?: https://github.com/kubernetes/community/blob/9406b4352fe2d5810cb21cc3cb059ce5886de157/contributors/devel/sig-instrumentation/logging.md?plain=1#L32 

I can change to V5 if you prefer though.

Reply:
Cool, I haven't seen that before. I think though the log levels used in this project are:
2 - errors, important msgs, 
4 - less important msgs
5 - debugging, tracing code flows

Reply:
Thanks for the info! Changed it to Level 5.